career/job_hunting.md

```diff
@@ -0,0 +1,75 @@
+When applying for new jobs your resume and your background will probably fall under three different categories:
+- Entry-level (about 0-3 years of experience)
+-  Career Change
+- Industry Experience
+
+This is important as you can narrow down your job search instead of applying for jobs where you would not be a good fit.
+
+Searching for a new job is an exhausting and difficult process but you can alleviate the process by answering two questions:
+
+- What kind of job do I want?
+- Understand what the employer wants
+
+### What kind of job do I want?
+
+You should know what you want and try to only search for roles that match those requirements.
+This makes it easier for you to find the right jobs and also du weed out the jobs that you don’t want and making sure that you keep focused.
+
+Before you start you job search decide on what you can’t compromise on, some guidelines are:
+
+##### - Role
+What is the job you want to do vs the job you could do
+##### Location
+Is relocation an option for you?
+
+##### - The company values
+What companies could you work for or absolutely not work for?
+
+#### Understand what the employer wants
+
+It is important to research before you send out an application so that you are sure that you know what the employer wants. Are your qualifications a good match for the company you are applying for or do they use the technologies that you are interested in?
```

Comment:
`company you are applying for`
`company you are applying to`

Reply:
Done 👍